match.go

```diff
@@ -5,44 +5,52 @@
 package validation
 
 import (
-	"errors"
 	"regexp"
 )
 
+// ErrValidationMatchInvalid is the error that returns in case of invalid format.
+var ErrValidationMatchInvalid = NewError("validation_match_invalid", "must be in a valid format")
```

Comment:
Could you rename it to `ErrMatchInvalid` so that it's consistent with other places?

Reply:
Sure.